packages/rx_bloc/lib/src/result_extensions.dart

```diff
@@ -12,12 +12,13 @@ extension ResultMapStreamX<E> on Stream<E> {
   /// ResultSuccess<int> result = Stream.value(10)
   ///   .mapToResult((value) => value * 10); // Result.success('100')
   /// ```
```

Comment:
Probably a comment with an example of using the methos with a tag can be provided here and for the other methods like:
  /// ResultSuccess<int> result = Stream.value(10)
  ///   .mapToResult((value) => value * 10, 'tag'); // Result.success('100', tag: 'tag')